booking.html

```diff
@@ -28,120 +28,118 @@
     <main>
```

Comment:
head is not completed. Missing data, such as link to js and style.css. you can just review other html page